CrossCutting/DopplerSapService/DopplerSapService.cs

```diff
@@ -66,7 +66,38 @@ public async Task<HttpResponseMessage> SendCurrency(IList<CurrencyResponse> curr
             }
             catch (Exception e)
             {
-                _logger.LogError(e, $"Error occurred trying to send information to Doppler SAP return http code {httpResponse.StatusCode}.");
+                _logger.LogError(e,
+                    $"Error occurred trying to send information to Doppler SAP return http code {httpResponse.StatusCode}.");
+                throw;
+            }
+
+            return httpResponse;
+        }
+
+        public async Task<HttpResponseMessage> SendUserBillings(IList<UserBilling> userBillingList)
+        {
+            var uri = _dopplerSapServiceSettings.Url;
+            _logger.LogInformation($"Building http request with url {uri}");
+
+            var httpRequest = new HttpRequestMessage
+            {
+                RequestUri = new Uri(uri),
+                Method = new HttpMethod("POST")
+            };
+            var requestContent = SafeJsonConvert.SerializeObject(userBillingList, _serializationSettings);
+            httpRequest.Content = new StringContent(requestContent, Encoding.UTF8);
+            httpRequest.Content.Headers.ContentType = MediaTypeHeaderValue.Parse("application/json; charset=utf-8");
+
+            var httpResponse = new HttpResponseMessage();
+            try
+            {
+                _logger.LogInformation("Sending request to Doppler SAP Api.");
+                httpResponse = await _httpClient.SendAsync(httpRequest).ConfigureAwait(false);
```

Comment:
I'm still thinking we should do something with this billing list. If something happens in between we won't know which data was sent to the DopplerSap service. Maybe comparing to SAP later on?
I think we should talk about this at the next meeting.